app/views/spree/admin/orders/_angular_pagination.html.haml

```diff
@@ -0,0 +1,17 @@
+.pagination
```

Comment:
I think this partial should be in /views/admin/shared. this is ready to be re-used, correct?